sharing/process/index.md

```diff
@@ -0,0 +1,79 @@
+
+Once we've verified the dataset meets [the qualifications]({% link about/policy/sgid/index.md %}#open-data-qualifications), you can share it with the appropriate Utah SGID ArcGIS Online group. All items in these groups are automatically added to Open Data. If this is your first time sharing a data with the particular group, you'll need to request access to the group.
+
+Datasets that languish in the staging group, or that are added to a Utah SGID group without approval, may be removed according to our [policies]({% link about/policy/sgid/index.md %}#reasons-for-removal).
+
+It is critical to provide complete metadata so that Open Data stays clean and easy to use. We've tracked down the sources of the information shown in the Open Data list view to help you understand where the information comes from:
+
+![An annotated Open Data item]({% link images/od_card_anno.png %}){: .flex .flex--center loading="lazy" }
+
+##### A Note about Open Data "Sources"
+
+> You should only have one "Source" for all the datasets coming from your organization. Esri explains where this value comes from [in this FAQ](https://doc.arcgis.com/en/hub/get-started/frequently-asked-questions.htm#GUID-9843B713-46D2-4938-A961-EC0CD81AE410). Basically, it first searches the ArcGIS Online item's metadata for an organization name. If it doesn't find anything there, it falls back to your ArcGIS organization name.
+>
+> To customize the source, we suggest setting it in the `Resource > Citation > Contact > Organization Name` metadata field. Otherwise, leave the metadata contact fields empty so it uses your ArcGIS Online organization name. We use this second way to show "Utah Automated Geographic Reference Center (AGRC)" as the source for all our datasets.
+>
+> To change the organization name in the ArcGIS Online item's metadata, you need to make sure your ArcGIS Online organization has [metadata enabled](https://doc.arcgis.com/en/arcgis-online/administer/configure-details.htm#ESRI_SECTION1_7FE1F060E03046E692BB36E5F6E3B2AE). Then you can use the Metadata editor on the ArcGIS Online item page to set either field:
+>
+>![Organization name sources in ArcGIS Online metadata]({% link images/od_agol_md.png %}){: .flex .flex--center .outline loading="lazy" }
+>
+> Depending on how you upload or overwrite your ArcGIS Online data, it may also overwrite the item's metadata with info from the source feature class metadata. We suggest you make any changes in both the feature class and ArcGIS Online metadata to avoid problems in the future. In the FGDC CSDGM style, the `Resource > Citation > Contact > Organization Name` field is in `Overview > Citation Contacts`. The `Resource > Contact > Organization Name` field is in `Resource > Points of Contact`.
+{: style="
+    font-size: 100%;
+    font-style: normal;
+    color: #111;
+    padding-left: 30px;
+    padding-right: 30px;
+    border-left: 2px solid #333;
+    "}
```

Comment:
**suggestion**: The vertical bar and the indentation seem a little awkward to me for such a large section of text.

Reply:
Yeah, I was trying for something of a "sidebar"-style content that was more an aside to the main text (in part to break up the huge blocks of text that this page became) but it does have its limitations. I'll see if I can find a better way to fit it in to the track-based rewrite.